### DISCUSSION TITLE
Hi , Is there any way to get the score of each character in paddle ocr?

### DISCUSSION THREAD

**Discussion by BuiPhamSonHa** [PaddlePaddle/PaddleOCR#12356] [Q&A]
Is there any way to get the score of each character in paddle ocr?

**Comment by changdazhou** [comment]
It is supported, but you need to modify the code here：https://github.com/PaddlePaddle/PaddleOCR/blob/0525f6bb01bfed401f767894619f6a25ee750892/ppocr/postprocess/rec_postprocess.py#L99

**Comment by itzananthakrishnan** [comment]
hi, any method for that
